spot-client/src/spot-remote/ultrasound/UltrasoundTransmitter.js

```diff
@@ -0,0 +1,128 @@
+import {
+    Transmitter,
+    loadDependencies,
+    profiles,
+    str2ab
+} from 'lib-quiet-js';
+
+/**
+ * Encapsulates the implementation details loading ultrasound and transmitting
+ * ultrasound messages.
+ */
+export default class UltrasoundTransmitter {
+    /**
+     * Calls to load any other files lib-quiet-js may need.
+     *
+     * @param {string} emscriptenPath - The url path, excluding the filename,
+     * for the emscripten used by lib-quiet-js.
+     * @param {string} memoryInitializerPath - The url path, excluding the
+     * filename, for the memory file dynamically loaded by the emscripten.
+     * @returns {void}
+     */
+    static loadDependencies(emscriptenPath, memoryInitializerPath) {
+        return loadDependencies({
+            emscriptenPath,
+            memoryInitializerPath
+        });
+    }
+
+    /**
+     * Initializes a new {@code ultrasoundTransmitter} instance.
+     *
+     * @param {number} transmissionDelay - The gap between emitting ultrasonic
+     * messages.
+     * @param {string} initialText - Any text to start transmitting immediately
+     * after instantiation.
+     */
+    constructor(transmissionDelay = 3000, initialText) {
```

Comment:
No mention of units for `transmissionDelay` (ideally in the variable name, but at least in the docs?)